.github/workflows/ci.yml

```diff
@@ -12,6 +12,10 @@ env:
   TURBO_TEAM: hashintel
   TURBO_REMOTE_ONLY: true
 
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
```

Comment:
This doesn't mess with the merge queue in some unexpected way, does it?

Reply:
We have this setup in the Rust CI and in the BP repository and I haven't noticed any issues with the merge queue. I don't think it should mess up the queue, as the queue has a branch per test.